packages/web3-shared/base/src/utils/resolver.ts

```diff
@@ -106,38 +106,55 @@ export function createExplorerResolver<ChainId, SchemaType, NetworkType>(
 
     return {
         explorerURL: getExplorerURL,
-        addressLink: (chainId: ChainId, address: string, tokenId?: string) =>
-            urlcat(getExplorerURL(chainId).url, addressPathname, {
+        addressLink: (chainId: ChainId, address: string, tokenId?: string) => {
+            const explorerURL = getExplorerURL(chainId)
```

Comment:
Why an unset explorer URL? Please set in the `NETWORK_DESCRIPTOR`.

Reply:
Because if `explorerURL.url` is `''`, then these method should return`''` rather than `/${path}/${address}` without domain